BraveWallet/Crypto/Stores/SwapTokenStore.swift

```diff
@@ -208,12 +210,16 @@ public class SwapTokenStore: ObservableObject {
       let swapParams = swapParameters(for: .perSellAsset)
     else { return }
     
+    isMakingTx = true
     rpcController.network { [weak self] network in
       guard let self = self else {
         self?.state = .error(Strings.Wallet.unknownError)
         self?.clearAllAmount()
+        self?.isMakingTx = false
```

Comment:
These are not needed, if this guard fails then self is nil and won't execute any of this